libcamera-sys/c_api/camera_manager.cpp

```diff
@@ -41,6 +43,16 @@ libcamera_camera_t *libcamera_camera_manager_get_dev(libcamera_camera_manager_t
     else
         return new libcamera_camera_t(camera);
 }
+#else
+libcamera_camera_t *libcamera_camera_manager_get_dev(libcamera_camera_manager_t *mgr, const char *dev) {
```

Comment:
Better to call *dev as id, and if you make it a const std::string &id (the same type as the camera manager exposes) you should get implicit conversion from const char * to const std::string I think ?

Reply:
I think we can not change the signature of libcamera_camera_manager_get_id, because it must be plain C99